src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Color[] COLORS = Color.values();
+    private static final Random RANDOM = new Random();
```

Comment:
Also about the naming of `Random` var, it may look like a constant because it is static and final here, but usually objects like that are dependencies that are initialized in the constructor or outside of that class. (which you can also do here)
On the other hand, constants are usually some primitives/wrappers/arrays or other data structures of things, etc.
So it can be named as just `random` and initialized in the constructor